docs/source/development/contributing.rst

```diff
@@ -152,7 +152,11 @@ The chart below is the expected release dates of minor releases.
 +------------+---------+
 | Date       | Version |
 +============+=========+
-| 02/10 2020 | 1.7.0   |
+| 03/08 2021 | 1.7.0   |
++------------+---------+
+| 04/30 2021 | 1.8.0   |
```

Comment:
@itholic can you remove 1.8.0 and 1.9.0? Once we port Koalas, we won't have more releases.